objc/UIViewController+PromiseKit.m

```diff
@@ -6,11 +6,13 @@
 #import <UIKit/UINavigationController.h>
 #import <UIKit/UIImagePickerController.h>
 #import "UIViewController+PromiseKit.h"
+@import MessageUI.MFMessageComposeViewController;
+@import MessageUI.MFMailComposeViewController;
```

Comment:
Ah, makes sense. I wasn't going to import originally but since the MessageCompose delegate method doesn't provide an error I needed to check the result value directly which left me with the option of either importing or assuming the int value of the Failed result.